uber/templates/forms/department/department_info.html

```diff
@@ -30,8 +30,9 @@
 
 <div class="row g-3 mb-3">
 {% block hours_panels %}
+    {# Consecutive hours has to be a whole number currently, so make python do whole number math here #}
 <div class="col-sm">{{ form_macros.input(department_info.max_consecutive_minutes,
-    value=(department_info.max_consecutive_minutes.data / 60 if department_info.max_consecutive_minutes.data else 0)) }}</div>
+    value=(department_info.max_consecutive_minutes.data // 60 if department_info.max_consecutive_minutes.data else 0), render_kw={'step': 1}) }}</div>
```

Comment:
This isn't the correct place to put the render_kw argument -- it needs to be put in the WTForms field definition in the backend. All this does is create the following HTML:
```
<input class="form-control" id="max_consecutive_minutes" name="max_consecutive_minutes" render_kw="{'step': 1}" type="number" value="9" data-keeper-edited="yes">
```

Since you're probably asleep, I'll go ahead and fix it.